_docs/featureBreakdown/v2.2-security.MD

```
@@ -0,0 +1,426 @@
+- Middleware:
+  - Continue to read token from **httpOnly cookie** (Next middleware can read it).
+  - Use that to guard `/admin` routes.
+- Remove `isAdmin.ts` usages that rely on reading token from `localStorage`.
+
+---
+
+## Change plan (frontend)
+
+---
+
+**Phase 1 – Stop storing JWT insecurely**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)